World.py

```diff
@@ -260,12 +260,15 @@ def resolve_random_settings(self):
         if self.settings.big_poe_count_random and 'big_poe_count' not in dist_keys:
             self.settings.big_poe_count = random.randint(1, 10)
             self.randomized_list.append('big_poe_count')
-        if self.settings.starting_tod == 'random' and 'starting_tod' not in dist_keys:
+        # If set to random in GUI, we don't want to randomize if it was specified as non-random in the distribution
+        if (self.settings.starting_tod == 'random' and 
+          ('starting_tod' not in dist_keys or self.distribution.distribution._settings['starting_tod'] == 'random')):
```

Comment:
not a fan of the indent :p

Is this available in `src_dict` or do we have to use `_settings`?

Reply:
Honestly, yeah, I was pretty unsure how to do that- How would you recommend to do it instead?

It is available in src_dict, I just wasn't sure which was preferred to use. I can easily switch it to use that instead.